bee-ledger/src/workers/storage.rs

```diff
@@ -111,45 +111,45 @@ impl<T> StorageBackend for T where
 {
 }
 
-pub(crate) fn insert_output_id_for_address_batch<B: StorageBackend>(
-    storage: &B,
-    batch: &mut <B as BatchBuilder>::Batch,
-    address: &Address,
-    output_id: &OutputId,
-) -> Result<(), Error> {
-    match address {
-        Address::Ed25519(address) => {
-            Batch::<(Ed25519Address, OutputId), ()>::batch_insert(storage, batch, &(*address, *output_id), &())
-                .map_err(|e| Error::Storage(Box::new(e)))
-        }
-        Address::Alias(_address) => {
-            todo!();
-        }
-        Address::Nft(_address) => {
-            todo!();
-        }
-    }
-}
-
-pub(crate) fn delete_output_id_for_address_batch<B: StorageBackend>(
-    storage: &B,
-    batch: &mut <B as BatchBuilder>::Batch,
-    address: &Address,
-    output_id: &OutputId,
-) -> Result<(), Error> {
-    match address {
-        Address::Ed25519(address) => {
-            Batch::<(Ed25519Address, OutputId), ()>::batch_delete(storage, batch, &(*address, *output_id))
-                .map_err(|e| Error::Storage(Box::new(e)))
-        }
-        Address::Alias(_address) => {
-            todo!();
-        }
-        Address::Nft(_address) => {
-            todo!();
-        }
-    }
-}
+// pub(crate) fn insert_output_id_for_address_batch<B: StorageBackend>(
```

Comment:
Reminder ✏️

Reply:
Yeah it will be reintroduced in a future PR when we handle the other output types, I commented it to avoid the warning